src/main/resources/assets/schemas/Dac.yaml

```diff
@@ -12,6 +12,8 @@ properties:
   email:
     type: string
     description: Optional DAC Email address
+  associatedDaa:
+    $ref: './DataAccessAgreement.yaml'
```

Comment:
Something about this isn't working - all schema entities are showing up with no values. I'm digging into this currently

Reply:
Ok, I think the correct magic here is to use the `allOf` syntax:
```
  associatedDaa:
    description: The Data Access Agreement selected for this DAC
    allOf:
      - $ref: './DataAccessAgreement.yaml'
```
The current syntax **_should_** work, but for some reason, breaks all of the other schemas. 

Additionally, we could use an entry in `api-docs.yaml` for the DAA:
```
    Data Access Agreement:
      $ref: './schemas/DataAccessAgreement.yaml'
```

Reply:
Hm, that's very weird— thank you for digging into that! I've implemented those changes.